descriptors/ModuleDescriptor-template.json

```diff
@@ -5,7 +5,7 @@
   "provides": [
     {
       "id": "remote-storage-configurations",
-      "version": "1.0",
+      "version": "1.1",
```

Comment:
I think this could be separate interface remote-storage-providers 1.0.

Reply:
Changed